### PR TITLE
fix(user-agent): remove dependency of

### PR DESCRIPTION
**Pull request by sebbsan**
#### Summary
remove dependency on unmaintained library

#### Description
used new `getVersion` method of `sphere-node-sdk`
removed dependeny on `load-module-pkg`
had to add `sphere-node-sdk` to main dependency - I guess that's the main debate here

#### Todo

- Tests
    - [ x] Unit
    - [ x] Integration
    - [x ] Acceptance
- [ ] Documentation

### CODE REVIEW

**Comment by sebbsan** [comment]
I guess PS people should review this since they are heavy users

File: src/coffee/helpers/user-agent.coffee [hunk 1]
```diff
@@ -1,9 +1,9 @@
 { arch, platform } = require 'os'
-getPkg = require 'load-module-pkg'
+{ SphereClient } = require 'sphere-node-sdk'
 
 module.exports = (pkgName, pkgVersion) =>
   throw new Error 'Package name and version is required' unless pkgName and pkgVersion
-  sdkPkgVersion = getPkg.sync('sphere-node-sdk').version
+  sdkPkgVersion = SphereClient.getVersion
```
Comment (daern91) [inline src/coffee/helpers/user-agent.coffee]: Im just wondering if we dont need to invoke this method? 
`SphereClient.getVersion()`

Comment (sebbsan) [reply src/coffee/helpers/user-agent.coffee]: right this can't work, even in CoffeeScript.. will fix it

Comment (sebbsan) [reply src/coffee/helpers/user-agent.coffee]: clearly the unit tests don't work either

**Pull request closed without merging**